uefi-mkconfig

```diff
@@ -293,13 +195,17 @@ main () {
         		partition_efis+="$(find "$partition_mount" -name "$kernel_image") "
 		done
 		
-		# Remove invalid entries
-		remove_uefi_entries
+		# Clear old entries for regeneration
+		for entry_number in {100..200}; do
+			efibootmgr -B -b $entry_number &> /dev/null
+		done
```

Comment:
I think this should be outside of the for loop. Otherwise, if there are multiple ESPs then we 1) delete all, 2) gen for ESP 1, 3) delete all again, 4) gen for ESP 2. End result is we only have entries for the last ESP.

Reply:
Yes, you are completely right. Will do that.